x/gamm/pool-models/stableswap/amm_test.go

```diff
@@ -100,3 +106,104 @@ func TestCFMMInvariantMultiAssets(t *testing.T) {
 		decApproxEq(t, k2, k3, kErrTolerance)
 	}
 }
+
+func (suite *StableSwapTestSuite) Test_StableSwap_CalculateAmountOutAndIn_InverseRelationship(t *testing.T) {
```

Comment:
Hi @mattverse how do i test this function? This is what i get when i try to run the command

![Uploading Screen Shot 2022-05-21 at 3.47.51 PM.png…]()

Reply:
so this is a part of a test suite, not a direct go test. I'd recommend using make file